src/hooks/useGetInvoice.ts

```diff
@@ -11,4 +12,16 @@ export const useGetInvoice = (uuid: string) => {
 
         return toInvoice(invoice)
     })
-}
\ No newline at end of file
+}
+
+export const useGetCalc = (uuid: string) => {
+    return useQuery(['calc', uuid], async () => {
+        const res = await fetch(`${BASE_URL}/invoices/calc/${uuid}`)
+
+        const invoiceCalcJson = await res.text()
+
+        const invoiceCalc: InvoiceCalc = JSON.parse(invoiceCalcJson)
```

Comment:
Is there a syntactically better way of doing this?  Do I even have to declare the type?  Seemed the downstream code in the tsx file responded moar better when I actually declared the type here, but it's javascript-land so maybe not necessary?